Fenced_Frames_Ads_Reporting.md

```diff
@@ -157,8 +157,7 @@ As mentioned in the explainer above, `reportEvent` beacons are POST requests and
 
 ### API to populate event data for reserved.top_navigation
 
```

Comment:
Let's also rephrase this [statement](https://github.com/WICG/turtledove/blob/main/Fenced_Frames_Ads_Reporting.md#:~:text=Distinguishing%20these%20flows%20here%2C%20since%20in%20principle%2C%20one%20should%20be%20able%20to%20report%20without%20the%20help%20of%20the%20other.) with
"Distinguishing these flows here, since in principle, one should be able to report without the help of the other but with an opt-in from the ad's origin for mainitaining web's security principles when it comes to origins."